client/src/components/forms/moodForm/InputValues.js

```diff
@@ -53,15 +53,15 @@ const InputValues = () => {
       setMeditationTime
     }
   } = moods;
+   const handleChange = (event) => {
+     const valueChange = event.target.name;
```

Comment:
event.target.name is not the function you call to set the value, it's the name of the input as specified here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#htmlattrdefname

you could do something like:
```
const [state, setState] = React.useState({});

const handleInputChange = e => setState(state => ({ ...state, [e.target.name]: e.target.value }));
...
return (
  <div>
    <input name="var1" value={state.var1} />
    <input name="var2" value={state.var2} />
  </div>
);
```

Reply:
And actually we've already set this up in your mood context

Reply:
See below in MoodSlider for how you should use your moodContext